config/samples/logging_v1_clusterlogging.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+spec:
+  logStore:
+    type: "elasticsearch"
+    elasticsearch:
+      nodeCount: 3
+      resources:
+        requests:
+          limits: 2Gi
```

Comment:
@lukas-vlcek @periklis thoughts on setting this to our recommended minimum default?  We are in this predicament of dev versus users assuming this is an OK limit for production